docs/overview.rst

```diff
@@ -233,8 +232,21 @@ information about the cluster. This includes:
 
 * **Cluster ID**: The unique ID of the cluster.
 
-* **Status**: The status of the cluster. This can be 'available', 'not
-  available', or 'setting up cluster'.
+* **Status**: Current status of your cluster:
+   
+   - GREEN: Your cluster is healthy.
```

Comment:
I think we don't really show these statuses in the UI, it's rather usually a combination of colour (green/yellow/red) and a description of what's happening, i.e. mappings here:

https://github.com/crate/cloud-ui/blob/4a4a8060b6a11fc2dad26a743ea42ef2e80068f1/src/components/ClusterHealth/ClusterHealth.jsx#L9

https://github.com/crate/cloud-ui/blob/4a4a8060b6a11fc2dad26a743ea42ef2e80068f1/src/components/ClusterHealth/messages.en.js#L3

Reply:
Hm, okay. I used the descriptions that were discussed [here](https://github.com/crate/cloud/issues/591#issuecomment-1298363501). 
Do you think I should just explain the Green/Yellow/Red then?

Reply:
I think so!